### PR TITLE
Task/id 367/lazy loading point incidents

### PR DESCRIPTION
**Pull request by saletG08**
Added the functionality that when pressing the checkbox will show the Street Incidents, and joined all the functionality of Natural Disasters and Traffic.

### CODE REVIEW

File: app/src/main/java/com/isc/hermes/MainActivity.java [hunk 1]
```diff
@@ -102,7 +102,7 @@ protected void onCreate(Bundle savedInstanceState) {
         initCurrentLocationController();
         initializeBurgerButtonToolBar();
         initializeFunctionalityOfTheBurgerButton();
-        setTheUserInformationInTheDropMenu();
+        //setTheUserInformationInTheDropMenu();
```
Comment (mmoralesv) [inline app/src/main/java/com/isc/hermes/MainActivity.java]: Why is this code commented? if some commented code is committed should be have a reason added or just remove commented code

File: app/src/main/java/com/isc/hermes/controller/IncidentsGetterController.java [hunk 2]
```diff
@@ -39,27 +43,34 @@ public IncidentsGetterController() {
...
 
     /**
      * Retrieves incidentsPointView near the specified location within a radius.
      *
-     * @param mapboxMap The Mapbox map instance.
      */
-    public void getNearIncidentsWithinRadius(MapboxMap mapboxMap, Context context) {
+    public void getNearIncidentsWithinRadius(Context context) {
         incidentsPointView = IncidentPointVisualizationController.getInstance(mapboxMap, context);
+        //mapboxMap.addOnCameraIdleListener(() -> loadNearIncidents(mapboxMap, context));
```
Comment (mmoralesv) [inline app/src/main/java/com/isc/hermes/controller/IncidentsGetterController.java]: Remove commented code

**Review by mmoralesv — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/isc/hermes/MainActivity.java (hunk 1 above)
Comment (saletG08) [reply app/src/main/java/com/isc/hermes/MainActivity.java]: Sorry trainer, there is a bug when initializing the app that appeared with a merge of yesterday night, it is related to the authentication management, I will put a comment and report an issue.

File: app/src/main/java/com/isc/hermes/controller/IncidentsGetterController.java (hunk 2 above)
Comment (saletG08) [reply app/src/main/java/com/isc/hermes/controller/IncidentsGetterController.java]: I'll remove it, I was waiting for the decision if we will continue with the lazy loading or not.

**Review by mmoralesv — APPROVED** [review]

**Pull request merged**